presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/file/FileHiveMetastore.java

```diff
@@ -310,6 +299,18 @@ else if (table.getTableType().equals(EXTERNAL_TABLE)) {
         return EMPTY_RESULT;
     }
 
+    protected void validateExternalLocation(Path externalLocation, Path catalogDirectory)
```

Comment:
move the protected functions to be below the public ones and in the order of being called